doc/languages-frameworks/vim.section.md

```diff
@@ -146,12 +146,27 @@ vim-full.customize {
 
 Note: this is not possible anymore for Neovim.
 
+## The updater script for Vim plugins {#the-updater-script-for-vim-plugins}
```

Comment:
this section seems redundant with `#updating-plugins-in-nixpkgs`. 
Might be best to move how to get vimPluginsUpdater over there.

Reply:
We need the section to reference it later by `#the-updater-script-for-vim-plugins`.